### PR TITLE
[css-shapes-1] propdef fixes

### PR DESCRIPTION
**Pull request by fantasai**
See https://github.com/w3c/csswg-drafts/issues/5160

### CODE REVIEW

**Comment by astearns** [comment]
Could you add this to the shape-margin propdef table, too?

**Comment by fantasai** [comment]
@astearns Done.

File: css-shapes-1/Overview.bs [hunk 1]
```diff
@@ -944,10 +944,10 @@ Float Area Shape: the 'shape-outside' property</h3>
 		Name: shape-outside
 		Value: none | [ <<basic-shape>> || <<shape-box>> ] | <<image>>
 		Initial: none
-		Applies to: floats
+		Applies to: floats and [=initial letter boxes=]
 		Inherited: no
 		Computed value: as <a href="#basic-shape-computed-values">defined</a> for <<basic-shape>> (with <<shape-box>> following, if supplied); else the computed <<image>>; else the keyword as specified
-		Animation type: as <a href="#basic-shape-interpolation">defined</a> for <<basic-shape>>, otherwise discrete
+		Animation type: by computed value; see <a href="#basic-shape-interpolation">interpolation</a> of <<basic-shape>>
```
Comment (astearns) [inline css-shapes-1/Overview.bs:950]: I think we need the discrete portion for shapes from image and/or combos of images/basic-shapes etc.

(thought I’d already left this comment, but now I don’t see it)

Comment (fantasai) [reply css-shapes-1/Overview.bs:950]: I think that's covered here: https://www.w3.org/TR/web-animations/#by-computed-value

Comment (astearns) [reply css-shapes-1/Overview.bs:950]: Image to image has the same type and number of components, so does not fit the requirements for discrete animation there.

Comment (fantasai) [reply css-shapes-1/Overview.bs]: Oh, right. I forgot that doesn't animate here the same way as normally. I'll drop that commit.

You might want to close https://github.com/w3c/csswg-drafts/issues/9009 as rejected.

**Pull request merged**